### PR TITLE
SQL create drop index minor fixes

### PR DESCRIPTION
**Pull request by kitaisreal**
### Changelog category (leave one):
- Not for changelog (changelog entry is not required)

Fix small inconsistencies. Follow up #35166.

### CODE REVIEW

**Comment by kitaisreal** [comment]
@Avogar did you have a change to look into changes ?

**Comment by Avogar** [comment]
> @Avogar did you have a change to look into changes ?

Not yet, let me look at it right now

**Review by Avogar — APPROVED** [review]

File: src/Parsers/ParserCreateIndexQuery.cpp [hunk 1]
```diff
@@ -87,18 +87,21 @@ bool ParserCreateIndexQuery::parseImpl(IParser::Pos & pos, ASTPtr & node, Expect
         return false;
 
      if (!parseDatabaseAndTableAsAST(pos, expected, query->database, query->table))
-         return false;
+        return false;
```
Comment (tavplubix) [inline src/Parsers/ParserCreateIndexQuery.cpp:90]: @kitaisreal, why did you remove space from here? Now there are 3 spaces, not 4 as it should be.

**Comment by tavplubix** [comment]
And if you are really interested in improving CREATE/DROP INDEX (not in removing spaces and rearranging lines of code), then you can take a look at #38643

File: src/Parsers/ParserCreateIndexQuery.cpp (hunk 1 above)
Comment (kitaisreal) [reply src/Parsers/ParserCreateIndexQuery.cpp:90]: In this file there is some issue with formatting. Follow up #38720.
